.github/workflows/generate-tag.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Gradle build
         uses: gradle/gradle-build-action@v2.2.0
         with:
-          arguments: --full-stacktrace assemble "-Psemver.scope=${{ github.event.inputs.scope }}" "-Psemver.stage=${{ github.event.inputs.stage }}"
+          arguments: --full-stacktrace build "-Psemver.scope=${{ github.event.inputs.scope }}" "-Psemver.stage=${{ github.event.inputs.stage }}"
```

Comment:
```suggestion
          arguments: --full-stacktrace build -Pkotlin.mpp.enableCompatibilityMetadataVariant=true "-Psemver.scope=${{ github.event.inputs.scope }}" "-Psemver.stage=${{ github.event.inputs.stage }}"
```

here too then

Reply:
`build` +  `-Pkotlin.mpp.enableCompatibilityMetadataVariant=true` does not work.

Since Kotest is released without `Pkotlin.mpp.enableCompatibilityMetadataVariant` we cannot resolve it's dependencies for `Pkotlin.mpp.enableCompatibilityMetadataVariant` :/

> We want to run the full build, with tests before we create the tag/release, right?
> The publish workflow is completely separate from this, so they shouldn't interfere with each other.